rules-tests/Arguments/Rector/ClassMethod/ArgumentAdderRector/Fixture/fixture4.php.inc

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Rector\Tests\Arguments\Rector\ClassMethod\ArgumentAdderRector\Fixture;
+
+class Value
+{
+    public function doSomething()
+    {
+
+    }
+}
+
+class Property
+{
+    public $value;
+}
+
+trait InteractsWithProperty
+{
+    public function doSomething(Property $property)
+    {
+        $property->value->doSomething();
+    }
+}
+
+final class SomeExampleClass
+{
+    use InteractsWithProperty;
+}
+
+?>
+-----
```

Comment:
this second part seems equal with above, you can remove it, and change fixture file with like `skip_mixed_scope.php.inc` and change class name to `class SkipMixedScope`

Reply:
fixed 🎉